README.md

```diff
@@ -71,6 +73,39 @@ linkStyle default opacity:0.5
   snaps_webpack_plugin --> snaps_rpc_methods;
   snaps_webpack_plugin --> snaps_sdk;
   snaps_webpack_plugin --> snaps_utils;
+  test_snaps --> examples/packages/background_events;
```

Comment:
Do we want this part of the diagram?

Reply:
I know, I'm just asking if we want to omit this part of the diagram

Reply:
Doesn't seem like an important part and takes up a bunch of space 😅

Reply:
Reverted the changes in this PR, seems out of scope anyway.